src/appBootstrapper.tsx

```diff
@@ -74,7 +91,7 @@ if (browser) {
 if (getBrowserWindow().navigator.webdriver) {
     $(document).ready(() => {
         $('body').addClass('e2etest');
-        window.e2etest = true;
+        ((window as any) as ICBioWindow).e2etest = true;
```

Comment:
I think we should switch to always use getBrowserWindow() and then make that return ICbioWindow which should inherit from browser window.  Of course, that may cause a host of other ts errors where we use getBrowserWindow() in the app.  But it would be an improvement.

Reply:
okay thats a good idea

Reply:
so getBrowserWindow is in commons, and a lot of things in ICBioWindow are not (e.g. ExtendedRouterStore, AppStore, IServerConfig). So refactoring this would mean bringing a lot of things into commons that aren't already. What do you think?

Reply:
i see.  yeah, that doesn't sound like a good idea.  maybe then just in this file make a variable browserWindow with this type and just refer to that?    i just think we should take the opportunity to make it cleaner since that's what this PR is all about

Reply:
sure, that works